docs/reference/edot-cloud-forwarder/aws.md

```diff
@@ -0,0 +1,323 @@
+
+| Log source | Description |
+| --- | --- |
+| VPC Flow | Logs generated by a VPC (Virtual Private Cloud). |
+| ELB Access | Logs generated by an ELB (Elastic Load Balancer). |
+| CloudWatch | Logs generated by AWS CloudWatch. |
+
+Read on to learn how to set up {{edot-cf}} for AWS.
+
+::::{important}
+{{edot-cf}} for AWS is not supported on {{ech}} or self-managed {{stack}}.
+::::
+
+## Prerequisites
+
+To collect logs using {{edot-cf}} for AWS you need:
+
+::::{tab-set}
+
+:::{tab-item} VPC Flow
+
+To collect VPC Flow logs, you need:
+
+- A VPC (Virtual Private Cloud).
+- An S3 bucket for storing flow logs.
+- A flow log configured with the S3 bucket as the destination.
+
+:::
+
+:::{tab-item} ELB Access
+
+To collect ELB (Elastic Load Balancer) Access logs, you need:
+
+- An ELB of any type (ALB, NLB, CLB).
+- An S3 bucket to store the access logs.
+- Access logging enabled, with the bucket as the destination.
+
+:::
+
+:::{tab-item} CloudWatch
+
+To collect CloudWatch logs, you need:
+
+- A CloudWatch Log Group where logs are stored.
+
+:::
+::::
+
+In addition, you need to know the URL of the managed OTLP endpoint and the API key for authentication.
+
+:::{include} ../_snippets/serverless-endpoint-api.md
+:::
+
+## Deployment considerations
+
+Before deploying {{edot-cf}} for AWS, keep these points in mind:
+
+- Deploy a separate CloudFormation stack for each log type, for example VPC Flow Logs, ELB Logs, or CloudWatch Logs. Each CloudFormation stack can only process one log source and format at a time.
+- Logs stored in S3 must be placed in separate buckets. Each log type should reside in its own dedicated bucket.
+
+## Download the template
+
+Download the CloudFormation template to deploy the appropriate stack based on your log source:
+
+| Log Source | CloudFormation template |
+| --- | --- |
+| S3 logs | `s3_logs-cloudformation.yaml` |
+| CloudWatch logs | `cloudwatch_logs-cloudformation.yaml` |
+
+## Configure the template
+
+{{edot-cf}} for AWS uses a CloudFormation template to deploy the EDOT Collector as a Lambda function.
+
+### Required settings
+
+These are the required settings you need to set in the CloudFormation templates:
+
+| Setting            | Description |
+| ------------------------- | --- |
+| `region`            | AWS region where the CloudFormation stack is to be deployed. |
+| `stack-name`        | Name of the CloudFormation stack. For example, `vpc-edot-cf`. Do not use the same name for different sources. |
+| `OTLPEndpoint`      | The OTLP endpoint URL used for data ingestion. |
+| `ElasticApiKey`     | API key for authentication with Elastic. |
+
+### Log source settings
+
+Set the following settings based on the log source:
+
+::::{tab-set}
+
+:::{tab-item} S3
+
+In the `s3_logs-cloudformation.yaml` template, set the following settings:
```

Comment:
@theletterf  won't this sentence make the users think that they need to edit the clouformation.yaml?

Reply:
Don't they? How are settings set?